docs/pages/versions/unversioned/sdk/filesystem.md

```diff
@@ -595,24 +595,146 @@ FileSystem.getTotalDiskCapacityAsync().then(totalDiskCapacity => {
 
 Returns a Promise that resolves to a number that specifies the total internal disk storage capacity in bytes, or JavaScript's [`MAX_SAFE_INTEGER`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/MAX_SAFE_INTEGER) if the capacity is greater than 2<sup>53</sup> - 1 bytes.
 
+## Storage Access Framework (**Android only**)
+
+The `StorageAccessFramework` is a namespace inside of the `expo-file-system` module, which encapsulates all functions which can be used with [SAF URIs](#saf-uri). You can read more about SAF in the [Android documentation](https://developer.android.com/guide/topics/providers/document-provider).
+
+## SAF URI
+
+A SAF URI is a URI that is compatible with the Storage Access Framework. It should look like this `content://com.android.externalstorage.*`. The easiest way to obtain such URI is by `askForDirectoryPermissionsAsync` method.
+
+## API
+
+```js
+import { StorageAccessFramework } from 'expo-file-system';
+```
+
+### Example Usage
+
+```ts
+import { StorageAccessFramework } from 'expo-file-system';
+
+// Requests permissions for external directory
+const permissions = await StorageAccessFramework.requestDirectoryPermissionsAsync();
+
+if (permissions.granted) {
+  // Gets SAF URI from response
+  const uri = permissions.directoryUri;
+
+  // Gets all files inside of selected directory
+  const files = await StorageAccessFramework.readDirectoryAsync(uri);
+  alert(`Files inside ${uri}:\n\n${JSON.stringify(files)}`);
+}
+```
+
+### `StorageAccessFramework.getUriForDirectoryInRoot(folderName)`
+
+Gets a [SAF URI](#saf-uri) pointing to a folder in the Android root directory. You can use this function to get URI for `StorageAccessFramework.requestForDirectoryPermissionsAsync` when you trying to migrate an album. In that case, the name of the album is the folder name.
```

Comment:
Do you think it's worth showing an example of this usage, either here or in the MediaLibrary docs?